### PR TITLE
Feature/fsr 46

### PR DESCRIPTION
**Pull request by nikiwycherley**
Add check boxes to key items and sync with polygon display

https://eaflood.atlassian.net/browse/FSR-46

### CODE REVIEW

File: server/src/js/components/map/outlook.js [hunk 1]
```diff
@@ -230,22 +235,27 @@ function OutlookMap (mapId, options) {
...
   //
 
-  // Outlook set day function
-  const setDay = (day) => {
-    // Set feature visibility
+  // Set feature visiblity
+  const setFeatureVisibility = () => {
     areasOfConcern.getSource().forEachFeature((feature) => {
-      const isVisible = parseInt(feature.get('day')) === parseInt(day)
+      const riskLevel = parseInt(feature.get('risk-level'))
```
Comment (teddmason) [inline server/src/js/components/map/outlook.js:248]: The parseInt method calls need to have the radix set to 10 as per answers here https://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix-parameter-when-calling-parseint this is to ensure that decimal numeration is used and parseInt doesn't make an incorrect assumption about the string being parsed

**Review by teddmason — CHANGES_REQUESTED** [review]
As per comment parseInts need their Radix setting to stop incorrect assumption by the method.

@JFSCH please can you do a review over this code as well, I'm not 100% confident in reviewing client side js, so good to get 2nd pair of eyes. thanks

**Review by JFSCH — COMMENTED** [review]
Code looks OK to me. Tested locally and looks fine.

File: server/src/js/components/map/outlook.js (hunk 1 above)
Comment (nikiwycherley) [reply server/src/js/components/map/outlook.js]: radix number added to parseInt function

**Review by teddmason — APPROVED** [review]

**Pull request merged**